src/oscontainer-deprecated-legacy-format.py

```diff
@@ -207,28 +204,22 @@ def oscontainer_build(containers_storage, tmpdir, src, ref, image_name_and_tag,
         subprocess.call(buildah_base_argv + ['rm', bid], stdout=subprocess.DEVNULL)
 
     if push:
-        print("Pushing container")
+        print("Pushing container to oci-archive")
```

Comment:
```suggestion
        print("Saving container to oci-archive")
```

A little more clear on what we're actually doing here.